addons/ocm-gateway-manager-addon/resources/cluster-gateway/v1.1.14/cluster-gateway-addon-manager/templates/addon-manager.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: cluster-gateway-addon-manager
-  namespace: open-cluster-management-addon
+  namespace: default
```

Comment:
what about vela-system?

Reply:
fixed, it should remain "open-cluster-management-addon" otherwise it will conflict w/ the existing installation from OCM